web/satellite/src/views/ErrorPage.vue

```diff
@@ -96,11 +96,23 @@ onMounted(() => {
         padding: 52px 24px;
         box-sizing: border-box;
         display: flex;
-        background: url('../../static/images/errors/dotWorld.png') no-repeat center 178px;
         flex-direction: column;
         justify-content: center;
         overflow-y: auto;
 
+        &::before {
+            content: "";
+            position: fixed;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            background: url("/static/static/images/errors/world.svg") no-repeat center;
```

Comment:
Although this path is functional at runtime, it produces an unresolved path error when building. It should be expressed relative to the current file: `../../static/images/errors/world.svg`.

Reply:
Thank you Jeremy!